README.md

```diff
@@ -1,27 +1,80 @@
 # ghshare
 
-> Encrypt data at rest using a user's public key on github.
+Encrypt data at rest using a user's public key on github.  
+
+Sometimes you want to share something encrypted and the recipient hasn't created a PGP key, but, they've uploaded a public key to GitHub. `ghshare` can fetch those keys and use them to encrypt a file.  
+
+## Installation
+
+### Download binary
+
+Download a [binary release](https://github.com/kjvalencik/ghshare/releases)  
+You will need to take the normal actions your platform requires to run a binary downloaded from the internet (macOS example: Enable macOS run unsigned binary, Make binary executable, Add binary to PATH, etc...)  
 
 ## Usage
 
 ```sh
-ghshare -h
+ghshare command [flags] [options]
 ```
 
-### GitHub Enterprise
+Available commands:  
 
-If your instance of GitHub Enterprise does not have a dedicated API host, you
-will need to suffix the host with `/api/v3`.
+### encrypt
 
-```sh
-ghshare encrypt -h https://example.com/api/v3 ...
+Basic form of encryption command:  
+
+```
+ghshare encrypt --input <plaintextFileName> --output <ciphertextFileName> --recipient <receiversGithubUsername>
+```
+
+> :information_source: Each of the receiver's GitHub public keys (available here: https://api.github.com/users/receiversGithubUsername/keys) will be used to encrypt the input file.
+> The receiver will only need to possess and use one of their corresponding private keys to decrypt the data.
+
+#### flags
+
+ flag                        | purpose
+-----------------------------|---------
+`--help`                     | Prints help information
+`--small`, `-s`              | Encrypt without header for small data, OpenSSL compatible
+`--version`, `-V`            | Prints version information
+
+#### options
+
+ flag               | param        | default                | required | purpose
+--------------------|--------------|------------------------|----------|---------
+`--host`, `-h`      | `<host>`     | https://api.github.com | `false`  | Endpoint to query for recipient's public keys
+`--input`, `-i`     | `<path>`     |                        | `true`   | Path to file to be encrypted
+`--output`, `-o`    | `<path>`     |                        | `true`   | File path to write encrypted data to
```

Comment:
Neither of these are required. When not provided, they will use `stdin` and `stdout` respectively.

Reply:
Fixed